build.sbt

```diff
@@ -905,9 +909,19 @@ lazy val kernelDefaults = (project in file("kernel/kernel-defaults"))
     Compile / classDirectory := target.value / "scala-2.13" / "kernel-defaults-classes",
 
     Test / javaOptions ++= Seq("-ea"),
-    // This allows generating tables with unsupported test table features in delta-spark
     Test / envVars += ("DELTA_TESTING", "1"),
+
+    // Put the shaded kernel-api JAR on the classpath (compile & test)
+    Compile / unmanagedJars += (kernelApi / Compile / packageBin).value,
+    Test    / unmanagedJars += (kernelApi / Compile / packageBin).value,
```

Comment:
We don't do this alignment formatting throughout `build.sbt` btw -- probably best to _not_ do it to match the existing style?

But I'm also okay if you think this formatting is better -- but then let's decide that we want to apply it elsewhere going forwards, too.